inc/namespace.php

```diff
@@ -27,7 +27,7 @@ function add_project( $type, $slug, $api_url ) {
 	add_filter(
 		'translations_api',
 		function ( $result, $requested_type, $args ) use ( $type, $slug, $api_url ) {
-			if ( $type === $requested_type && $slug === $args['slug'] ) {
+			if ( $type . 's' === $requested_type && $slug === $args['slug'] ) {
```

Comment:
Based on the supported types in https://github.com/WordPress/wordpress-develop/blob/6b366c6620fdd5960cedcdf80955966a715efa82/src/wp-admin/includes/translation-install.php#L15 this looks good.